documentation/API-design-guidelines.md

```diff
@@ -1287,44 +1290,54 @@ response:
       "ipv4Addr": "192.168.0.1"
     },
     "uePort": 5060,
-    "eventType": "ROAMING_STATUS"
+    "type": "org.camaraproject.device-status.ROAMING_STATUS"
   },
   "eventSubscriptionId": "456g899g",
   "startsAt": "2023-03-17T16:02:41.314Z",
   "expiresAt" : "2024-03-17T00:00:00.000Z"
 }
 ```
 
-Note: If the API provides both patterns (indirect and resource-based), and the API customer requests both (instance based + subscription), the 2 requests should be handled independently & autonomously. Depending on server implementation, it is acceptable, when the event occurrs, that one or two notifications are sent to listener.
+Note: If the API provides both patterns (indirect and resource-based), and the API customer requests both (instance based + subscription), the 2 requests should be handled independently & autonomously. Depending on server implementation, it is acceptable, when the event occurs, that one or two notifications are sent to listener.
 
 
 ### 12.2 Event notification
 
 The event notification endpoint is used by the API server to notify the API consumer that an event occured.
 
-Note: The notification is the message posted on listener side. We describe the notification message in the CAMARA OAS but it could confusing because this endpoint should be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notifications.
+CAMARA event notification leverages **[CloudEvents](https://cloudevents.io/)** as it is a vendor-neutral specification for defining the format of event data. A generic neutral CloudEvent notification swagger is available in commonalities/artifact directory (notification-as-cloud-event.yaml).
+
+Note: The notification is the message posted on listener side. We describe the notification(s) in the CAMARA API using the `callbacks`. From API consumer it could confusing because this endpoint must be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notifications.
 
-Only Operation POST is provided for `eventNotifications` and the expected response code is `204`.
+Only Operation POST is provided for `eventNotifications` and the expected response code is `204`. 
+The url for this `POST` operation must be specified in the swagger as `{$request.body#/webhook/notificationUrl}`. 
 
-For consistence among CAMARA APIs the uniform `eventNotifications` model must be used:
+For consistence among CAMARA APIs the uniform CloudEvents model must be used with following rules:
```

Comment:
For consistency across CAMARA APIs the uniform CloudEvents model must be used with following rules:

Reply:
Sorry, I can't locate the "notification-as-cloud-event.yaml" file within the "commonalities/artifact" directory. I'm uncertain if I've overlooked something.

Reply:
Is it possible for us to suggest a pattern for the "ID," whether it should be a UUID or another option?

Reply:
@sachinvodafone  "notification-as-cloud-event.yaml" file is subject to another PR https://github.com/camaraproject/Commonalities/pull/43 .

Currently it is behind the changes we are discussing here in Design Guidelines.

Reply:
> Is it possible for us to suggest a pattern for the "ID," whether it should be a UUID or another option?
@sachinvodafone : Sure, could you kindly make a suggestion before the next commonalities, so that people have time to react to your suggestions before the meeting?